trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests/src/org/eclipse/tracecompass/incubator/trace/server/jersey/rest/core/tests/stubs/TmfConfigurationSourceTypeStub.java

```diff
@@ -66,33 +71,32 @@ public TmfConfigurationSourceTypeStub(@JsonProperty("id") String id,
             builder.setConfigParamDescriptors(parameterDescriptors.stream().map(stub -> stub.getConfig()).collect(Collectors.toList()));
         }
         fConfig = builder.build();
+        fSchema = schema;
     }
 
+    /**
+     * @return the JSON schema string
+     */
+    public JsonNode getSchema() {
```

Comment:
Should this not be added to ITmfConfigurationSourceType interface? It seems to be called by the serializer, although as getSchemaFile() returning a File?

Reply:
This would that TmfConfigurationSourceType would require the jackson JSON parser because `fSchema` is of type `JsonNode'.